cmd/cloudshell_open/cloudrun_test.go

```diff
@@ -50,3 +54,52 @@ func Test_tryFixServiceName(t *testing.T) {
 		})
 	}
 }
+
+// this integration test depends on gcloud being auth'd as someone that has access to the crb-test project
+// todo(jamesward) only run the integration test if the environment can do so
+func Test_describe(t *testing.T) {
+	tests := []struct {
+		project string
+		region string
+		service string
+		wantErr bool
+	}{
+		{"crb-test", "us-central1", "asdf1234zxcv5678", true},
+		{"crb-test", "us-central1", "cloud-run-hello", false},
+	}
+	for _, tt := range tests {
+		t.Run(tt.service, func(t *testing.T) {
+			_, err := describe(tt.project, tt.service, tt.region)
+
+			tparams := fmt.Sprintf("describe(%s, %s, %s)", tt.project, tt.region, tt.service)
+
+			if err == nil && tt.wantErr {
+				t.Error(tparams + " was expected to error but did not")
+			}
+
+			if err != nil && !tt.wantErr {
+				t.Error(tparams + " produced an error: %s", err)
+			}
+		})
+	}
+}
+
+// this integration test depends on gcloud being auth'd as someone that has access to the crb-test project
```

Comment:
let's delete this test as well
if you want to really test parsing, I suggest we add testdata/example-svc.yaml and we can parse that.